### PR TITLE
Ci tool param

### PR DESCRIPTION
**Pull request by jbradl11**
Setting up basic Travis testing of running `make all all-r4 CI_TOOL=true`, using the updated FHIR-Patient-Generator.

### CODE REVIEW

**Review by okeefm — CHANGES_REQUESTED** [review]
Changes make sense to me, given that the `Makefile` can now install FBC on its own. PR needs a rebase/deconfliction before it can get merged.

File: travis/test_measures.sh [hunk 1]
```diff
@@ -6,6 +6,8 @@ if ! ./travis/rebuild_bundles.sh; then
 fi
 
 cd fhir-patient-generator
+git checkout ci-param-tool-rebase
+make all all-r4 CI_TOOL=true
```
Comment (okeefm) [inline travis/test_measures.sh]: Do you actually need both the `all` and `all-r4` targets here? I think they do the same thing in FHIR Patient Generator.

File: travis/test_measures.sh [hunk 2]
```diff
@@ -6,6 +6,8 @@ if ! ./travis/rebuild_bundles.sh; then
 fi
 
 cd fhir-patient-generator
+git checkout ci-param-tool-rebase
```
Comment (tstrass) [inline travis/test_measures.sh]: Did you mean to change this to `master`? That other branch was merged and deleted.

**Review by tstrass — CHANGES_REQUESTED** [review]

File: travis/test_measures.sh (hunk 2 above)
Comment (tstrass) [reply travis/test_measures.sh]: Oh wait no `checkout master` would be redundant, so I guess did you want to remove this?

File: travis/test_measures.sh (hunk 1 above)
Comment (okeefm) [reply travis/test_measures.sh]: Also I just realized my comment could be confusing-- I believe that `make all` and `make all-r4` are aliases for the same makefile target in FHIR Patient Generator, is what I meant to say.

**Review by tstrass — APPROVED** [review]

**Review by okeefm — APPROVED** [review]
Looks good to me. Thanks for the updates!

**Pull request merged**